utils/spdx/src/test/kotlin/SpdxExpressionTest.kt

```diff
@@ -407,8 +412,7 @@ class SpdxExpressionTest : WordSpec() {
             "not contain duplicate valid choice for a complex expression" {
                 "(a OR b) AND (a OR b)".toSpdx().validChoices() should containExactlyInAnyOrder(
                     "a".toSpdx(),
-                    "b".toSpdx(),
-                    "a AND b".toSpdx()
+                    "b".toSpdx()
```

Comment:
Esp. @MarcelBochtler, to double-check, do you agree with this to have been an invalid choice?

Reply:
Why? `(a OR b) AND (a OR b) == (a OR b)`, so `b` should be a valid choice.

Reply:
Sorry, i completely misunderstood the comment and now it's too late because it's already merged. My interpretation would be that if there are two different locations which offer the choice `(a OR b)` it would technically be possible to choose `a` for the first and `b` for the second. However, I'm not sure if this is a scenario that we would want to support.

Reply:
> two different locations which offer the choice `(a OR b)`

You're right: The important bit here is that you need to take the *location* of the licenses into account (e.g. outbound vs inbound). I'll create a revert PR.

Reply:
See https://github.com/oss-review-toolkit/ort/pull/4946.